test/Harrington/removeScripts.js

```diff
@@ -0,0 +1,72 @@
+const fs = require("fs");
```

Comment:
Is there a reason you're keeping this file around? I don't see it being required anywhere.

Reply:
Yes, it's a utility that can be used to strip an HTML file of all its <script> elements. I found it's best to defang sample HTML by removing all <script> elements -- then there will be no odd, unexpected loading from the original server while running a test. A number of times, I found the page I was scraping suddenly changed underneath me, invalidating everything. (Maybe it was bad coding, e.g., not targeting the pageService at the right place for injecting test data, but whatever was causing it, defanging the HTML eliminated such surprises.)

Keeping it around may be useful to maintainers in the future, or provide a blueprint for others using sample HTML for testing. 

Since you asked whether there's a reason for keeping it, I'm wondering if the Javadoc for the `execute()` function is clear enough as to its purpose.